### ISSUE TITLE
Netz

### ISSUE THREAD

**Issue by zenozeng** [zenozeng/cocolour#11]
https://github.com/nickewing/netz

Netz is a Clojure implementation of a multilayer perceptron (MLP), a type of feedforward artificial neural network. Netz provides functions for training and running MLPs. Training is accomplished via gradient descent batch Rprop or standard backpropagation.
